### DISCUSSION TITLE
Firebase Identity Platform TOTP (with Authenticator) availability

### DISCUSSION THREAD

**Discussion by billypap1** [firebase/firebase-ios-sdk#11632] [Q&A] [answered]
Hello, the TOTP is production ready?
As I read in the docs, it is in pre-GA and has been integrated to all platforms (web, Android,iOS). 
Can we use the TOTP in our apps across all platforms or will there be breaking changes in future versions of the SDKs?

**Comment by renkelvin** [comment]
Hi, the feature is currently in preview. We will update doc(https://cloud.google.com/identity-platform/docs/admin/enabling-totp-mfa) once it's GA.

**Reply by billypap1** [reply]
@renkelvin I checked the docs (https://cloud.google.com/identity-platform/docs/admin/enabling-totp-mfa) and the feature is not in preview anymore, so is it ready for production?

**Reply by renkelvin** [reply, marked as answer]
@billypap1 Yes, totp is in GA now https://cloud.google.com/identity-platform/docs/release-notes#September_21_2023
